src/internal/_equals.js

```diff
@@ -1,114 +1,196 @@
+var _containsWith = require('./_containsWith');
 var _arrayFromIterator = require('./_arrayFromIterator');
 var _functionName = require('./_functionName');
 var _has = require('./_has');
 var identical = require('../identical');
 var keys = require('../keys');
 var type = require('../type');
 
+module.exports = (function() {
+  /**
+   * private _uniqContentEquals function
+   * Notes:
+   * 1. this function will return true on [1,2,3] === [1,1,1]
```

Comment:
my bad - wrong order taken. 
```javascript
_uniqContentEquals([1,1,1], [1,2,3], [], []) // => true
```
I've changed the comment to make it more explanatory